### DISCUSSION TITLE
Share your ideas to make pwa-asset-generator better!

### DISCUSSION THREAD

**Discussion by onderceylan** [onderceylan/pwa-asset-generator#478] [Ideas]
## 👋 Happy to see you here!
We’re using **Ideas** discussions as a place to share ideas and connect with other members of our community. We hope that you:
  * Share your ideas without hesitation.
  * Help each other shape a new idea.
  * React on an idea to make it more visible.

Thanks for doing this for our community!

**Comment by djhmateer** [comment]
Hi - https://news.ycombinator.com/item?id=25520655 has some ideas in it to reduce the number of images. I've not dug into the veracity.

**Reply by onderceylan** [reply]
Hi @djhmateer, thanks for sharing. I will have a look.
